onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -738,6 +739,8 @@ public void onLocationChanged(Location l) {
         if (firstFix) {
             // If its our first location (across all BaseMapFragments), move the camera to it
             setMyLocation(true, false);
+            //Notify the onFocusChangeListener when we get the first location
+            onFocusChanged(null,null,l);
```

Comment:
@barbeau I add this to notify the onFocusChange Listener in the first time we get a location.